google/cloud/bigtable/tools/generate_rpc_policy_parameters.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Generate header file for retry policy defaults from GAPIC yamls
```

Comment:
I think it is customary to format like this:

```py
"""
Generate header file for retry policy defaults from GAPIC yamls
```

And now that I read this more carefully:

```
Generates a header file configuring the default retry policy parameters from the GAPIC YAML config files.
```

Reply:
fixed.